auth/src/main/java/com/firebase/ui/auth/ui/email/RegisterEmailFragment.java

```diff
@@ -99,21 +99,20 @@ public View onCreateView(LayoutInflater inflater,
 
         View v = inflater.inflate(R.layout.register_email_layout, container, false);
 
-        mPasswordFieldValidator = new PasswordFieldValidator(
```

Comment:
Yup to both of your comments since I had to downgrade the gradle plugin, otherwise lint will complain.